webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/campaigns/DataTypes.scala

```diff
@@ -53,15 +53,30 @@ trait Campaign {
 }
 
 case class CampaignInfo (
-  id : CampaignId
+    id : CampaignId
   , name : String
   , description : String
   , status : CampaignStatus
   , schedule : CampaignSchedule
   , duration: Duration
 )
 
-case class CampaignId (value : String, rev: Revision = GitVersion.DEFAULT_REV)
+case class CampaignId (value : String, rev: Revision = GitVersion.DEFAULT_REV) {
```

Comment:
I'm not sure we need a revision for campaign id, but if we need one for futur-proveness, it needs to be serialized with revision